apps/web/src/composables/ethers.ts

```diff
@@ -31,6 +31,75 @@ export default function useEthers() {
     }
   }
 
+  /**
+   * Estimate gas fee using EIP 1559 methodology
+   * @returns string in ETH
+   */
+  async function estimateEIP1559GasFee(rpcUrl: string, unsignedTransaction: ethers.utils.Deferrable<ethers.providers.TransactionRequest>) {
```

Comment:
Usually TS infers the type so it's not a problem for the caller but I think we should still add the return types.
```ts
  async function estimateEIP1559GasFee(...): string
```

Reply:
Sweet. Thanks, Howie. Refactored and done. 👍